lizrd/text/datasets.py

```diff
@@ -79,10 +79,22 @@ def _get_random_wiki_example(self) -> str:
 class C4Dataset(AbstractDataset):
     total_gpt2_tokens = 173_648_052_806  # number of tokens in the C4 dataset when using GPT2TokenizerFast
 
-    def __init__(self, seed: Optional[int] = None, split: str = "train"):
+    def __init__(
+        self,
+        seed: Optional[int] = None,
+        split: str = "train",
+        use_dummy_dataset: bool = False,
+    ):
         super().__init__(seed=seed)
         assert split in ["train", "validation"]
-        self.dataset = load_dataset("c4", "en", split=split)
+        if use_dummy_dataset:
+            if split != "train":
+                raise NameError(
+                    "Dummy dataset only supports train split for C4 dataset"
+                )
+            self.dataset = load_dataset("stas/c4-en-10k", split=split)
```

Comment:
A może dajmy, że jak używamy dummy datasetu to train set i validation set to to samo? I tak na tym nic nie trenujemy, chodzi o to, żeby kod działał.